src/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import 'bootstrap/dist/css/bootstrap.css';
 import ReactDOM from 'react-dom/client';
+import 'bootstrap/dist/css/bootstrap.css';
```

Comment:
haha why have you re-added the same line that is already there

Reply:
fixed it!